tests/python/test_api_ui_vizio_actions.py

```
@@ -134,6 +141,20 @@ def test_vizio_toggle_endpoint_turns_on(tmp_path: Path) -> None:
     assert tv.calls == ["on"]
 
 
+def test_vizio_toggle_endpoint_returns_409_for_unresponsive_tv(tmp_path: Path) -> None:
+    tv = _FakeVizioTv("192.168.1.10", "Kitchen TV", is_on=True, unresponsive=True)
+    state = _state(vizio_tvs=[tv], cache_path=tmp_path / "cache.sqlite")
+    client, app = _client()
```

Ruff (0.16.1)

[warning] Unpacked variable `app` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)